pkg/model/coordinator/milestones.go

```diff
@@ -88,7 +88,7 @@ func createCheckpoint(trunkHash trinary.Hash, branchHash trinary.Hash, mwm int,
 }
 
 // createMilestone creates a signed milestone bundle.
-func createMilestone(seed trinary.Hash, index milestone.Index, securityLvl int, trunkHash trinary.Hash, branchHash trinary.Hash, mwm int, merkleTree *merkle.MerkleTree, powFunc pow.ProofOfWorkFunc) (Bundle, error) {
+func createMilestone(seed trinary.Hash, index milestone.Index, securityLvl consts.SecurityLevel, trunkHash trinary.Hash, branchHash trinary.Hash, mwm int, merkleTree *merkle.MerkleTree, powFunc pow.ProofOfWorkFunc) (Bundle, error) {
 
 	// get the siblings in the current Merkle tree
 	leafSiblings := siblings(index, merkleTree)
```

Comment:
This is no longer needed, as it directly corresponds to `merkleTree.AuditPath(uint32(index))`

Reply:
Oh, didn't see that :)
Thx